prototype/src/main/webapp/upcoming-meetings.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>GoMeet Upcoming Meetings</title>
+  <script src="script.js"></script>
+  <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1 style="text-align:center;">GoMeet Upcoming Meetings</h1>
+    <ul>
+      <h4 style="text-align:center;"><a href="view-meeting.html">[Meeting Name 1]</a></h3>
+      <h4 style="text-align:center;"><a href="view-meeting.html">[Meeting Name 2]</a></h3>
+      <h4 style="text-align:center;"><a href="view-meeting.html">[Meeting Name 3]</a></h3>
```

Comment:
For these `h4`s consider using a class for all of these, then add this text-align style into the css file. This means that if you want to change how the list of meetings looks, you don't have to change all of these elements.

Reply:
Done! :)

Reply:
This means that every `<h4>` will be centered - if that's what you want that's ok, but if you only want these ones to be centered, you could put `class="foo"` in the h4 tag and then have the css rule apply to `.foo`.

Reply:
Changed it to class="center" and replaced the h4 styling with a "center" class